custom/icds/messaging/indicators.py

```
@@ -108,6 +127,37 @@ def location_names_from_user_id(self, user_ids):
         return {self.awc_locations[l] for l in loc_ids}
 
 
+class AWWAggregatePerformanceIndicator(AWWIndicator):
+    template = 'ls_aggregate_performance.txt'
+
+    def get_value_from_fixture(self, fixture, attribute):
+        xpath = './rows/row[@is_total_row="False"]'
+        rows = fixture.findall(xpath)
+        for row in rows:
+            owner_id = row.find('./column[@id="owner_id"]')
+            if owner_id.text == self.user.sql_location.name:
+                return row.find('./column[@id="{}"]'.format(attribute)).text
```

i think we'd also want it to fail hard if the AWC is not found here